dotnet/src/SemanticKernel.Abstractions/PromptTemplate/InputVariable.cs

```diff
@@ -33,4 +33,13 @@ public sealed class InputVariable
     /// </summary>
     [JsonPropertyName("is_required")]
     public bool IsRequired { get; set; } = true;
+
+    /// <summary>
+    /// JsonSchema describing this variable.
+    /// </summary>
+    /// <remarks>
+    /// This string will be deserialised into an instance of <see cref="KernelJsonSchema"/>.
```

Comment:
Nit: deserialized (I realize both are valid spellings in different geographies, but JsonSerializer uses a 'z', so it's nice for consistency :-)